includes/class-wc-payments-incentives-service.php

```diff
@@ -153,15 +153,17 @@ public function fetch_connect_incentive(): ?array {
 
 		if ( 200 === wp_remote_retrieve_response_code( $response ) ) {
 			// Decode the results, falling back to an empty array.
-			$results = json_decode( wp_remote_retrieve_body( $response ), true ) ?? [];
+			$results = json_decode( wp_remote_retrieve_body( $response ), true );
```

Comment:
I noticed we added the new `if( ! empty( $results ) )` check, but what gets returned by the function in that case? It may be worth explicitly returning early or something similar in the case that the result isn't what we expect.

Reply:
We will return the initial value `$incentive = []` plus the `ttl` logic below, so it can be cached for the given amount of time. Am I missing something? 🤔

Reply:
Chatted about it over Slack, just wanted to confirm that response was expected. 🙂